src/init

```diff
@@ -63,6 +64,10 @@ function exit_handler()
     exit 254
   fi
 
+  if [[ $CHILD_PROCESS = "PLUGIN_INITIALIZER" ]]; then
+    exit 255
+  fi
+
```

Comment:
Please remove these lines. I will add error handling in another PR.

Reply:
> Please remove these lines. I will add error handling in another PR.

Removed